data/brands/shop/electronics.json

```
@@ -80,6 +80,17 @@
         "short_name": "Apple"
       }
     },
+    {
+      "displayName": "Automatic Centre",
+      "id": "automaticcentre-775c70",
+      "locationSet": {"include": ["ph"]},
+      "tags": {
+        "brand": "Automatic Centre",
+        "brand:wikidata": "Q4826504",
+        "name": "Altex",
```

```suggestion
        "name": "Automatic Centre",
```

Maybe you copy this entry and forget to replace name value？

https://github.com/osmlab/name-suggestion-index/blob/e82636d4811e4972ee3e116f566a41f9e0958419/data/brands/shop/electronics.json#L44-L51